pkg/admin/backup.go

```diff
@@ -0,0 +1,50 @@
+package admin
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/openshift/microshift/pkg/config"
+
+	"k8s.io/klog/v2"
+)
+
+var (
+	cpArgs = []string{
+		"--verbose",
+		"--recursive",
+		"--preserve",
+		"--reflink=auto",
+	}
+)
+
+func MakeBackup(target, name string) error {
+	if err := config.CreateDir(target); err != nil {
+		klog.Errorf("Failed to create dir %s: %v", target, err)
+		return err
+	}
+
+	dest := filepath.Join(target, name)
+	tmp_dest := dest + "-tmp"
+
+	args := append(cpArgs, config.DataDir, tmp_dest)
```

Comment:
What happens here if the destination folder already exists?

Reply:
That's a good catch. Do you think that error in that case is good for now? I'd like to not add another param (`--force`) right now (maybe later for non-ostree systems)

Reply:
If we get an error, that would be OK. I suspect we will end up with another copy of the directory inside the existing one.

Reply:
I didn't explain it properly... I want to add an error if the path already exists (so there's neither overwriting nor copying into)

Reply:
That sounds like a good plan, yes.